x/participationrewards/types/msgs.go

```diff
@@ -17,22 +18,43 @@ func NewMsgSubmitClaim(
 	}
 }
 
-// GetSigners Implements Msg.
+// GetSigners implements Msg.
 func (msg MsgSubmitClaim) GetSigners() []sdk.AccAddress {
 	fromAddress, _ := sdk.AccAddressFromBech32(msg.UserAddress)
 	return []sdk.AccAddress{fromAddress}
 }
 
-// ValidateBasic Implements Msg.
+// ValidateBasic implements Msg: stateless checks.
 func (msg MsgSubmitClaim) ValidateBasic() error {
 	errors := make(map[string]error)
 	if _, err := sdk.AccAddressFromBech32(msg.UserAddress); err != nil {
 		errors["UserAddress"] = err
 	}
 
-	// TODO: check for valid zone (chainID)
+	if msg.Zone == "" {
```

Comment:
not a problem, just wondered if there was a reason for `== ""` here rather than using `len()`

Reply:
yes len() is a better approach